recipes/default.rb

```diff
@@ -21,6 +22,55 @@
 
 include_recipe 'erlang'
 
+
+## NB - conf files must be created before the RabbitMQ service first starts.
+##      Otherwise if a static nodename is specified it triggers a restart of
+##      the service but rabbitmqctl can't find the existing one and the
+##      restart (& your Chef run) noisily fail.
+
+if node['rabbitmq']['logdir']
+  directory node['rabbitmq']['logdir'] do
+    owner 'rabbitmq'
+    group 'rabbitmq'
+    mode '775'
+    recursive true
+  end
+end
+
+directory node['rabbitmq']['mnesiadir'] do
+  owner 'rabbitmq'
+  group 'rabbitmq'
+  mode '775'
```

Comment:
We've changed it to `750` in RabbitMQ packages recently because some users can't have a world-readable data directory.